x-pack/plugins/security_solution/cypress/screens/create_new_rule.ts

```diff
@@ -127,6 +127,8 @@ export const MITRE_ATTACK_ADD_TECHNIQUE_BUTTON = '[data-test-subj="addMitreAttac
 
 export const MITRE_ATTACK_ADD_SUBTECHNIQUE_BUTTON = '[data-test-subj="addMitreAttackSubtechnique"]';
 
+export const PREVIEW_HEATHER_SUBTITLE = '[data-test-subj="header-panel-subtitle"]';
```

Comment:
Small typo I think:
```suggestion
export const PREVIEW_HEADER_SUBTITLE = '[data-test-subj="header-panel-subtitle"]';
```

Reply:
Totally right, changing it right now, thanks :)